code/modules/mob/living/holders.dm

```diff
@@ -187,3 +187,10 @@
 /obj/item/weapon/holder/animal/slime/attack_self(mob/user)
 	..()
 	unfreeze()
+
+/obj/item/weapon/holder/animal/pillow
+	name = "pillow holder"
+	desc = "Comforbable"
```

Comment:
:b:

Reply:
>comfortable pillbug
>comf-orb

This may have actually been intentional.

Reply:
![photoshop_2018-10-03_19-29-33](https://user-images.githubusercontent.com/40967382/46443159-ab451380-c742-11e8-90bb-bdccd31fb82f.png)